static/app/views/integrationPipeline/githubInstallationSelect.tsx

```diff
@@ -0,0 +1,151 @@
+import {Fragment, useState} from 'react';
+import styled from '@emotion/styled';
+import {Observer} from 'mobx-react';
+import * as qs from 'query-string';
+
+import {addLoadingMessage} from 'sentry/actionCreators/indicator';
+import {BaseAvatar} from 'sentry/components/core/avatar/baseAvatar';
+import {Button} from 'sentry/components/core/button';
+import type {SelectKey, SelectOption} from 'sentry/components/core/compactSelect';
+import {CompactSelect} from 'sentry/components/core/compactSelect';
+import {Tooltip} from 'sentry/components/core/tooltip';
+import Form from 'sentry/components/forms/form';
+import FormModel from 'sentry/components/forms/model';
+import List from 'sentry/components/list';
+import ListItem from 'sentry/components/list/listItem';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+type Installation = {
+  avatar_url: string;
+  id: number;
+  login: string;
+};
```

Comment:
This is the info/props we expect the receive from the Pipeline.

Reply:
would it be more secure to also expect to receive some kind of code? so we can send it back to the pipeline? 🤔 cc @mdtro